teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization)
+    super(45, 'gala', parent_permission: true)
+    @specialization = specialization
+  end
```

Comment:
- [ ] with the previous comment I meant that the parameters need to be added to the arguments the initialize method will accept, this will make it look something like below:
```
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
```

Reply:
`parent_permission = true` this code use as parameter gives error based on research `parent_permission: true` is advised to be used for boolean arguments, kindly refer me where that works, thanks